polyfill/lib/absolute.mjs

```diff
@@ -16,39 +16,33 @@ import {
   NANOSECONDS
 } from './slots.mjs';
 import { absolute as STRING } from './regex.mjs';
+import bigInt from 'big-integer';
 
 export class Absolute {
   constructor(epochNanoseconds) {
-    if ('bigint' !== typeof epochNanoseconds) throw RangeError('bigint required');
-    const epochMilliseconds = Number(epochNanoseconds / BigInt(1e6));
-    const restNanoseconds = Number(epochNanoseconds % BigInt(1e6));
+    if (('bigint' !== typeof epochNanoseconds) && !bigInt.isInstance(epochNanoseconds)) throw RangeError('bigint required');
```

Comment:
If native `BigInt` support is not available, we are assuming the user would use the same `BigInt` polyfill as us. Is that a very good assumption? Shouldn't we *not* assume the existence of a library, take primitives and construct an instance using the library ourselves? I know that defeats half the purpose, so feel free to correct me on that, but it still helps us avoid convoluted math while also not imposing our choice of polyfill onto the user.

Reply:
No we don't. Which is why your comments below are off. 

That's precisely NOT the assumption, which is why we NEVER expos one of the potentially polyfilled BigInts through the API

Reply:
`!bigInt.isInstance` 🤦‍♂ can we make sure we never talk about this, ever?

Reply:
not sure why my comment didn't post from before :-/

this *is* exposed because a user-provided object will behave differently if it's built using this polyfill or not, precisely because of the `bigint.isInstance` check.